src/pages/Vault/index.tsx

```diff
@@ -123,30 +120,23 @@ const Vault = (): JSX.Element => {
       try {
         // TODO: should update using `react-query`
         const [
-          vault,
           feesPolkaBTC,
           lockedAmountBTC,
           collateralization,
           apyScore,
           issuableAmount
         ] = await Promise.allSettled([
-          window.bridge.vaults.get(vaultAccountId, COLLATERAL_ID_LITERAL),
```

Comment:
This was updated with react-query. Actually, this is out of scope of this feature. I did it unintentionally but the update did not have any problem and could be considered an improvement. I did not revert it. I fully tested it and it works.

Reply:
No problem, that's fine - it's only slightly out of scope :) thanks for flagging though.